src/main/preload/mattermost.js

```diff
@@ -7,21 +7,18 @@
 /* eslint-disable no-magic-numbers */
 
 import {ipcRenderer, webFrame} from 'electron';
-import log from 'electron-log';
 
 import {NOTIFY_MENTION, IS_UNREAD, UNREAD_RESULT, SESSION_EXPIRED, SET_SERVER_NAME, REACT_APP_INITIALIZED, USER_ACTIVITY_UPDATE} from 'common/communication';
 
 const UNREAD_COUNT_INTERVAL = 1000;
 const CLEAR_CACHE_INTERVAL = 6 * 60 * 60 * 1000; // 6 hours
 
-Reflect.deleteProperty(global.Buffer); // http://electron.atom.io/docs/tutorial/security/#buffer-global
```

Comment:
Why is this being removed?

Reply:
Cherry-pick issue

Reply:
it is failing stating that global.buffer doesn't exist anymore, also tryng that url doesn't exist anymore 🤷

Reply:
I think it was used by the `remote` module, which we don't use anymore.